content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -35,6 +35,12 @@ Before you begin, make sure you've already [installed and configured the Agent][
 
 Datadog has a series of pluggable packages which provide out-of-the-box support for instrumenting a series of libraries and frameworks. A list of these packages can be found in the [Compatibility Requirements][1] page. Import these packages into your application and follow the configuration instructions listed alongside each [Integration][1].
 
+### Automatic Compile-Time Instrumentation (Developer Preview)
+
+Instead of manually instrumenting your apps with the [Integrations][1], you can use [orchestrion][6] to automatically add instrumentation to your application, including its dependencies and the Go standard library, at compilation time. This ensures maximum instrumentation coverage in your app (eg. across your dependencies), and moves instrumentation code out of the way, so you can focus on your application logic and keep your existing code as it is.
+
+Follow the instructions listed on the [orchestrion documentation][6]
+
```

Comment:
Present an either/or choice here; turning both ways in collapsible sections, editing the lede to present choices succinctly.